packages/server/src/GraphQLCache.js

```diff
@@ -246,9 +242,15 @@ export class GraphQLCache implements GraphQLCacheInterface {
     rootDir: string,
     includes: string[],
   ): Promise<Array<GraphQLFileMetadata>> => {
+    let pattern: string;
+    if (includes.length === 1) {
```

Comment:
Nit: add link to issue

Reply:
I don't think it's a bug. It sort of depends on which implementation of globs you look at. Some of them (fish shell) expanded `a{p}e` into `ape` and others (bash, sh, minimatch) leave `a{p}e` as is. My guess is that minimatch is modeled after bash and filing a ticket will be fruitless.

Reply:
I just mean we should add a note-to-self about why `length ===1` is special cased (i.e. a link to #222). To prevent future engineers from cleaning up this code.

Reply:
Ah, no problem. I'll update the PR with a comment.